service/dap/server.go

```diff
@@ -1800,10 +1800,21 @@ func (s *Session) onThreadsRequest(request *dap.ThreadsRequest) {
 			if g.Thread != nil && g.Thread.ThreadID() != 0 {
 				thread = fmt.Sprintf(" (Thread %d)", g.Thread.ThreadID())
 			}
+			var labels strings.Builder
+			if len(g.Labels()) > 0 {
+				var keys []string
+				for k := range g.Labels() {
+					keys = append(keys, k)
+				}
+				sort.Strings(keys)
+				for _, k := range keys {
+					fmt.Fprintf(&labels, " %s:%s", k, g.Labels()[k])
+				}
+			}
```

Comment:
Can we rewrite it as follows:
```suggestion
			keys := make([]string, 0, len(g.Labels()))
			for k := range g.Labels() {
				keys = append(keys, k)
			}
			sort.Strings(keys)
			var labels strings.Builder
			for _, k := range keys {
				labels.WriteByte(' ')
				labels.WriteString(k)
				labels.WriteByte(':')
				labels.WriteString(g.Labels()[k])
			}
```
?

Reply:
The implementation has changed after the discussion in #3493, as we don't show all labels any more.

I'm curious though, why is writing individual bytes manually better than Fprintf to a builder?